portfolio/src/main/webapp/inport.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Same note as above - this file should probably be called "import.html", not "inport.html".